dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/pipeline/outbound/HeaderHandler.java

```diff
@@ -27,6 +27,8 @@
 import io.netty.handler.codec.http.HttpResponseStatus;
 import io.netty.handler.codec.http.HttpUtil;
```

Comment:
In my opinion there are enough changes here to add a date range to the copyright header that includes 2025